src/auth.ts

```diff
@@ -303,8 +297,12 @@ const getAuth = function (
 	 * });
 	 */
 	async function getUserId(): Promise<number> {
```

Comment:
I'm wondering whether now that whoami returns the whole result, it would make sense to drop this :thinking: 
Alternatively if the argument could be that this saves us from repeating the `if not user actor then throw` check, then it might make sense to also add a `getUsername` method, but I'm not really enthusiastic about that...

Reply:
@thgreasi Well the good thing about having getUsername is that it would be a very similar replacement to the current `whoami` which can make the transition from v17 -> v18 slight easier. Why you deslike the idea?

Reply:
That was what I was thinking as well @otaviojacobi .
I'm fine with it just:
* afraid that ppl might just replace whoami w/ getUsername w/o taking the time to think whether there is a better alternative
* not loving the amount of separate special methods that we have for each field of the user whoami.

Let's add it if you are more convinced than me.

Reply:
@otaviojacobi how about `getUserInfo` or `getUserDetails`?

Reply:
@thgreasi both work for me, I think getUserInfo is slighlt more clear than details